main.go

```diff
@@ -76,6 +76,9 @@ var (
 	// filesystemScanIncludePaths = filesystemScan.Flag("include-paths", "Path to file with newline separated regexes for files to include in scan.").Short('i').String()
 	// filesystemScanExcludePaths = filesystemScan.Flag("exclude-paths", "Path to file with newline separated regexes for files to exclude in scan.").Short('x').String()
 
+	fileScan     = cli.Command("file", "Scan a file (defaults to standard in)")
+	fileScanPath = fileScan.Flag("path", "Path of the file to scan").Default(os.Stdin.Name()).String()
```

Comment:
**Thought:** It would make more sense to me to put file scanning under the `filesystem` subcommand. In general I think the command should look like `trufflehog filesystem FILE_OR_DIR [FILE_OR_DIR...]` but I know we have the `--directory` flag. Maybe we can start with a repeatable `--file` flag?

Reply:
I don't think it necessarily needs to match enterprise (the CLIs are already pretty divergent and have opportunity to improve), but what about stdin in this case? Maybe File + Directory are covered under Filesystem, and Stdin becomes its own subcommand?

Probably a separate discussion, but if we were to redo this, what would the preferred terminology be that is more technically accurate and human friendly?